INSTALL.md

```diff
@@ -57,7 +57,7 @@ Shogun is part of [homebrew-science](https://github.com/Homebrew/homebrew-scienc
     $ sudo brew install shogun
 
 ###Windows <a name="windows"></a>
-We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC. It is recommended to use " Visual Studio 14 2015 " which has a good support for C++11, or you can only use " MSBuild " to build Shogun. Run CMD as administrator with the following commands：
+We currently do not support a Shogun binary on Windows platform. Shogun natively compiles under Windows using MSVC. It is recommended to use " Visual Studio 14 2015 " or " MSBuild " to build Shogun which both have good support to C++11. Run CMD as administrator with the following commands：
```

Comment:
support `for` and **use**, i.e. \*\*\*\* for emphasizing things.

"we currently do not provide a Shogun binary for Windows platform."